tests/functional/spec/features/submit_page/submit_page_collapsible_summary.spec.js

```diff
@@ -30,8 +30,8 @@ describe("Collapsible Summary", () => {
       expect($(SubmitPage.collapsibleSummary()).getText()).to.contain("Property Details");
       expect($(SubmitPage.collapsibleSummary()).getText()).to.contain("House Details");
 
-      expect($(SubmitPage.insuranceAddressQuestion()).isDisplayed()).to.be.false;
-      expect($(SubmitPage.numberOfPeopleQuestion()).isDisplayed()).to.be.false;
+      expect($(SubmitPage.insuranceAddressQuestion()).getText()).to.contain("");
```

Comment:
These tests will always pass cause `""` is always there so you need to check for something else (or lack of existence of something else.